x-pack/plugins/ml/public/file_datavisualizer/components/results_view/results_view.js

```diff
@@ -5,65 +5,59 @@
  */
 
 
-import React, {
-  Component,
-} from 'react';
+import React from 'react';
 
 import {
   EuiHorizontalRule,
   EuiTabbedContent,
+  EuiButton,
+  EuiSpacer,
 } from '@elastic/eui';
 
 import { FileContents } from '../file_contents';
 import { Summary } from '../summary';
 import { FieldsStats } from '../fields_stats';
 
-export class ResultsView extends Component {
-  constructor(props) {
-    super(props);
+export function ResultsView({ data, results, showEditFlyout }) {
 
-    this.state = {};
-  }
+  console.log(results);
 
```

Comment:
Nit - just making sure we want to log the results here... :)

Reply:
I'm leaving log statements in for the time being as this is still on a feature branch and very much a work in progress.